Cargo.lock

```diff
@@ -4857,6 +4893,33 @@ dependencies = [
  "urlencoding",
 ]
 
+[[package]]
+name = "google-cloud-bigquery"
+version = "0.7.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "c48abc8687f4c4cc143dd5bd3da5f1d7ef38334e4af5cef6de4c39295c6a3fd0"
+dependencies = [
+ "anyhow",
+ "arrow 50.0.0",
```

Comment:
We introduce full arrow 50 dependencies here. We should verify whether it will greatly affect the compile time of DEBUG and RELEASE build. If yes, we may need to consider how to avoid this dependency given that we technically don't use `arrow` in `google-cloud-bigquery`.

Reply:
| branch  | time | size  | build num  |
| ------------- | ------------- | ------------- | ------------- |
| release main  |  27m 24s  |27173092317  | 1471  |
| release pr  | 28m 14s  |27275146355  |1476  |
| debug main   |  4m 31s   | 48776880726  | 1413  |
| debug pr   |4m 51s     |48920738495   |1418  |